science/apbs/Portfile

```diff
@@ -41,12 +41,13 @@ use_parallel_build no
 configure.optflags      -O3 -ffast-math
 configure.args-append   -DENABLE_OPENMP:BOOL=OFF \
                         -DENABLE_BEM=OFF \
+                        -DCMAKE_CXX_FLAGS_RELWITHDEBINFO:STRING="-I${worksrcpath}/include -O2 -g -DNDEBUG -std=c++98" \
```

Comment:
You use -O2 optimisation here, but the line above is selecting -O3 instead. Which should the build use ?

Reply:
I set the CXX_FLAGS_RELWITHDEBINFO flags to those currently be set in the build. Note that the current cmake build does a poor job of transferring includes down to the compile lines and using CMAKE_CXX_FLAGS_RELWITHDEBINFO was the only way I could find to get the include for the headers passed. I would rather not mess further with the build and instead move on to porting the newer 3.0.0 release later which should have much improved cmake support.

Reply:
If you don’t want the build to use `-O3 -ffast-math ` then why set these flags in `configure.optflags ` ?

Reply:
I didn't add that.

[https://github.com/macports/macports-ports/commit/da7b18b678144793d38a81c19c8cd71a3220f040#diff-feac28b79ae2f0c4429d43f0539579d5326e3bd39778bb27ce639016d53e2e64](url)

Reply:
Feel free to change the -O2 to -O3 -ffast-math when you commit.

Reply:
I am not going to change it. For one its not possible easy the github ui to make changes like that as you merge. You are also the maintainer so its not unreasonable to ask you why the port has these inconsistent settings.

Reply:
Since pymol is unlikely to support ABPS tools again, feel free to switch this package to unmaintained.

Reply:
If you wish to make that change you should submit a MR to do so. It is no one else's job to do the work for you.